site/_docs/history.md

```diff
@@ -28,23 +28,50 @@ For a full list of releases, see
 Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
-<!--
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.34.0">1.34.0</a> / XXXX-XX-XX
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.34.0">1.34.0</a> / 2023-03-XX
 {: #v1-34-0}
 
-This release comes x months after [1.33.0](#v1-33-0),
-contains contributions from x contributors, and resolves x issues.
+This release comes 1 month after [1.33.0](#v1-33-0),
+contains contributions from 18 contributors, and resolves 34 issues. It adds roughly 15 new functions
+in BigQuery library for handling dates, times, and timestamps and provides a fix
+([<a href="https://issues.apache.org/jira/browse/CALCITE-5522">CALCITE-5522</a>]) for a small breaking
+change in `DATE_TRUNC` function ([<a href="https://issues.apache.org/jira/browse/CALCITE-5447">CALCITE-5447</a>]),
+which was introduced accidentally in [1.33.0](#v1-33-0). Among other small improvements and fixes it
+brings a new breaking change ([<a href="https://issues.apache.org/jira/browse/CALCITE-3870">CALCITE-3870</a>])
+changing the default sub-query expansion strategy of `SqlToRelConverter`. 
 
 Contributors to this release:
-x,
-y (release manager),
-z.
+Alessandro Solimando,
+Benchao Li,
+Brandon Chong,
+Dmitry Sysolyatin,
+Francis Chuang,
+Gian Merlino,
+Guillaume Massé,
+Jiajun Xie,
+Julian Hyde,
+Moritz Mack,
+Oliver Lee,
+Peng Wang,
+Stamatis Zampetakis (release manager),
+Tanner Clary,
+Tim Nieradzik,
+TJ Banghart,
+xinqiu.hu,
+Zou Dan.
+
+<!--
+Command used to generate the release notes
+git log b64cb1325cfe1a5143ea3ca534f991b6f881c3c5..ee9b80b0b68d442991dfaa142722e3488ec73e79 --format="%s" | sed 's|\[\(CALCITE-[0-9]\+\)\]|* [<a href="https://issues.apache.org/jira/browse/\1">\1</a>]\n|'git log b64cb1325cfe1a5143ea3ca534f991b6f881c3c5..ee9b80b0b68d442991dfaa142722e3488ec73e79 --format="%s" | sed 's|\[\(CALCITE-[0-9]\+\)\]|* [<a href="https://issues.apache.org/jira/browse/\1">\1</a>]\n|'
+-->
 
 #### Breaking Changes
 {: #breaking-1-34-0}
 
-As of [CALCITE-3870](https://issues.apache.org/jira/browse/CALCITE-3870),
-the default value of
+[<a href="https://issues.apache.org/jira/browse/CALCITE-3870">CALCITE-3870</a>]
```

Comment:
it seems strange to split the description of 3870 over two paragraphs

Reply:
Fixed in https://github.com/apache/calcite/pull/3103/commits/561070e3e0e4e232d0d4dd8d046c89c823a566d0